sdk/iot/core/inc/az_iot_core.h

```diff
@@ -21,12 +21,16 @@
 /**
  * @brief Azure IoT service MQTT connection properties.
  *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
  */
+
 enum
 {
-  AZ_CLIENT_DEFAULT_MQTT_CONNECT_PORT = 8883,
-  AZ_CLIENT_DEFAULT_MQTT_CONNECT_CLEAN_SESSION = 0x20,
```

Comment:
why was this removed, I was using it?

Reply:
It's not portable to a lot of the MQTT libraries. Most of them control clean session with a boolean flag and not the actual bit within the `CONNACK` message.
Also, it's not clear what the default meant. The guidance is to always have CleanSession=false. (see https://github.com/Azure/azure-sdk-for-c/pull/658/files#diff-19fef307f19572073d4a230f548b6b41R72 within the MQTT state machine MD file updated in this PR.)